ansible/playbooks/host-setup.yml

```diff
@@ -40,3 +40,31 @@
   environment: "{{ deployment_environment_variables | default({}) }}"
   roles:
     - host_setup
+
+- hosts: nova_compute_nodes
```

Comment:
Not sure if that is a custom group I would have attached it to openstack_compute_nodes group

Reply:
Pulled nova_compute_nodes from the inventory file, what advantages will openstack_compute_nodes group bring?